src/components/DataTable/DataTablePagination/DataTablePagination.stories.js

```diff
@@ -91,7 +90,7 @@ const render = ({
             <TableSelectRow {...getSelectionProps({ row })} />
             {row.cells.map(cell => (
               <TableCell key={cell.id}>
-                {cell.value ? cell.value : missingDataCharacter}
+                {cell.value ? cell.value : '-'}
```

Comment:
Perhaps we can add the '-' as a static `missingDataCharacter` property to the `DataTable` to still provide some consistency here - That way, here they could say `{cell.value ? cell.value : DataTable.missingDataCharacter}`

Reply:
Thanks @SimonFinney -- what kind of consistency are you seeking here? I just added it to the stories, since it's like the preferred missing data character for us. But other developers in other contexts may want a different character to represent missing data. The point of this line is to provide developers an example (along with the example demo as a whole) so that they know what is possible. I'm not currently seeing the value of adding a status property for a single character that they'd likely change anyway?

Reply:
It's about offloading the work for those that want to remain compliant - For example:

- https://github.ibm.com/xforce-exchange/xfe-isc/blob/master/services/reports/src/common/XFEDataTable.js#L113
- https://github.ibm.com/investigate/investigate/blob/master/src/components/tableWrapper/TableWrapper.js#L152

I assume these consumers aren't interested in what character to use for the `missingDataCharacter`, only that it's in line with what we recommend - if for example, this was to change, they won't need to keep an eye on Storybook or revisit it. It's a low-hanging helper for something that provides a lot of maintenance value.

Reply:
Thanks for your thoughts!

I said "preferred by us" above not to reference some official recommendation (which I am not aware of), but instead to reference that this is merely the way I'm presenting the story demo of the component.

The stakes are low for them to just copy & paste the `-` character in a ternary inside this demo, or go a different direction if they prefer. I don't think we should save off a single character like this as a static property (and then add that to our growing list of exports). 👍

Reply:
I'm not sure I entirely agree - I'm not hugely concerned about what's exported as long as it provides added value in the form of consistency and efficiency to consumers. That's the whole point of why we're doing this 🙃

We can defer this for now, but I'll reach out to a selection of consumers currently leveraging this to understand their needs and we can create a follow-up appropriately.